### PR TITLE
Fix lack of space in fatal error ("tofile an issue")

### PR DESCRIPTION
**Pull request by strk**
Also reduces duplication by storing the error message in a variable

### CODE REVIEW

**Comment by untitaker** [comment]
FWIW you can preserve the line wrapping, since all that is missing is a space at the end of each line:

    "lorem ipsum"
    "dolor"

becomes:

    "lorem ipsum "
    "dolor"

File: khal/khalendar/event.py [hunk 1]
```diff
@@ -638,15 +638,13 @@ def __init__(self, *args, **kwargs):
         try:
             starttz = getattr(self._vevents[self.ref]['DTSTART'].dt, 'tzinfo', None)
         except KeyError:
-            logger.fatal(
-                "Cannot understand event {} from calendar {},\n you might want to"
-                "file an issue at https://github.com/pimutils/khal/issues"
-                "".format(kwargs.get('href'), kwargs.get('calendar'))
-            )
+            msg = "Cannot understand event {} from " \
+                  "calendar {}, you might want to file an issue at " \
```
Comment (hobarrera) [inline khal/khalendar/event.py]: Intentation should remain the same (also, as a convention, indentation should always be four spaces).

File: khal/khalendar/event.py [hunk 2]
```diff
@@ -638,15 +638,13 @@ def __init__(self, *args, **kwargs):
         try:
             starttz = getattr(self._vevents[self.ref]['DTSTART'].dt, 'tzinfo', None)
         except KeyError:
-            logger.fatal(
-                "Cannot understand event {} from calendar {},\n you might want to"
-                "file an issue at https://github.com/pimutils/khal/issues"
-                "".format(kwargs.get('href'), kwargs.get('calendar'))
-            )
+            msg = "Cannot understand event {} from " \
```
Comment (hobarrera) [inline khal/khalendar/event.py]: Prefer parenthesis to slashes to join lines like these.

File: khal/khalendar/event.py [hunk 3]
```diff
@@ -638,15 +638,13 @@ def __init__(self, *args, **kwargs):
...
+                  "calendar {}, you might want to file an issue at " \
+                  "https://github.com/pimutils/khal/issues" \
+                  .format(kwargs.get('href'), kwargs.get('calendar'))
+            logger.fatal(msg)
             raise FatalError(  # because in ikhal you won't see the logger's output
-                "Cannot understand event {} from calendar {},\n you might want to"
```
Comment (hobarrera) [inline khal/khalendar/event.py]: Was there any downside to simply adding the space here?

**Comment by strk** [comment]
Downside: duplicated string, would have had to fix in two places.

See how you like 4f1063c136a99a28210cf736bf7c259c793cf544

**Comment by geier** [comment]
Thank you!

**Comment by hobarrera** [comment]
Duh, sorry, the diff confused my and I failed to noticed the duplication. Cool, thanks!

**Pull request merged**